controller/controller.go

```diff
@@ -377,12 +376,7 @@ func (c *Controller) resourceSet(obj interface{}) (*ResourceSet, error) {
 	}
 
 	if len(found) == 0 {
-		accessor, err := meta.Accessor(obj)
-		if err != nil {
-			c.logger.Log("level", "warning", "message", "cannot create accessor for object", "object", fmt.Sprintf("%#v", obj), "stack", fmt.Sprintf("%#v", err))
-		} else {
-			c.logger.Log("level", "debug", "message", "no resource set for reconciled object", "object", accessor.GetSelfLink())
-		}
+		c.logger.Log("level", "debug", "message", "no resource set for reconciled object", "object", fmt.Sprintf("%#v", obj))
```

Comment:
This breaks how we create `object` key. So far it was always a value from `SelfLink`.
Also with the link you can check the object, so we don't have to make additional noise it in the logs.

Do you have some use case when it's necessary to have that logged?

Reply:
You cannot just check the self link when your CI does not just allow that. My intention was to make the questionable object more visible. Or at least its values. I would actually be interested in the configured resource sets but they do not have names and our objects have version bundle versions but not all have that. Don't know. Just tried to improve this somehow but I see the issues.

Reply:
Also: Would it make sense to use `warning` or `error` level for this log entry?

Reply:
Not really since you expect this somehow when there are multiple controllers but differently configured.

Reply:
In normal circumstances we have access to the cluster. Maybe we can print the object in a nice form at the beggining of the test in e2e-harness?

Reply:
This might be an option. I close this here. Thanks.